sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -126,6 +126,13 @@ def test_recur_cond_workflow(self):
     from .testdata.tekton_loop_dsl import pipeline
     self._test_pipeline_workflow(pipeline, 'tekton_loop_dsl.yaml')
 
+  def test_loop_with_numeric_workflow(self):
+    """
+    Test compiling a loop with literal separator in workflow.
```

Comment:
*with numeric range

This test does covert start-end, but what about start-step-end scenario?

Reply:
I added one more example for start, end, step.